### DISCUSSION TITLE
How do I enable delay global setting in django

### DISCUSSION THREAD

**Discussion by neoneo40** [codingjoe/django-select2#273] [Q&A]
```python
widgets = {
            'cars': CarWithMultipleWidget(
                attrs={
                    'data-ajax--delay': 250,
                }
            ),
        }
```

- A delay like this must be provided here to avoid excessive overload on the server. How do I set this globally?
- You should not refer to https://select2.org/data-sources/ajax#rate-limiting-requests here. Please tell me how I can set this globally in django. It's a pain to set each widget one by one, and changing it later is also a headache.

**Comment by codingjoe** [comment]
Hi @re4lfl0w,

Thanks for reaching out. There isn't a way to set it globally unless you extend the JavaScript implementation we provided. This package doesn't use too many settings. I'd encourage you to use inheritance. You can build a widget mixin class, that sets the appropriate delay and make sure it's used everywhere, where this delay is applicable. Thus avoiding code duplication.

I hope that answer helps.

Cheers,
Joe
